pkg/healthcheck/healthcheck.go

```diff
@@ -111,8 +116,9 @@ type checker struct {
 	// when the check is executed
 	description string
 
-	// hintURL provides a pointer to more information about the check
-	hintURL string
+	// hintAnchor, when appended to `HintBaseURL`, provides a URL to more
+	// information about the check
+	hintAnchor string
```

Comment:
TIOLI: What are your thoughts on renaming this to `hintSection`, to me, the name `hintAnchor`contains HTML implementation detail which may not be necessary in this case.

Reply:
hm, imo the name `hintAnchor` is very descriptive as to what it is, and it seems helpful for someone who is not familiar with the check code to know exactly how that piece fits in with the hint page

Reply:
heh, ok i get both perspectives. i'm going to revert back to `hintAnchor`, as i do have a slight preference for communicating that this is intended as a URL anchor.